tutorials/sequencing/centralized.md

```diff
@@ -0,0 +1,118 @@
+
+![Rollkit with Centralized Sequencer](/public/img/sequencing.jpg)
+
+## Prerequisites {#prerequisites}
+
+This tutorial serves as a comprehensive guide for using the [centralized-sequencer](https://github.com/rollkit/centralized-sequencer) with your chain. 
+
+Before proceeding, ensure that you have completed the [quick start](/tutorials/quick-start) or [build a chain](/tutorials/wordle) tutorial, which covers installing the rollkit CLI, building your chain, and running your chain.
+
+:::tip
+It is important to note that the centralized sequencer expects a DA layer to be running when it starts. This means that you need to launch your DA Layer before starting the centralized sequencer.
+:::
+
+Additionally, you should have completed one of the [DA Layer tutorials](../da/overview) and have your DA layer ready to use with the centralized sequencer. This tutorial will use the [local-da layer](../da/local) as an example.
+
+## Installation the Centralized Sequencer {#installation-centralized-sequencer}
+
+```sh-vue
+git clone --depth 1 --branch {{constants.centralizedSequencerLatestTag}} https://github.com/rollkit/centralized-sequencer.git
+cd centralized-sequencer
+make build
+./build/centralized-sequencer -h
+```
+
+```sh
+Usage:
+  -host string
+    	centralized sequencer host (default "localhost")
+  -port string
+    	centralized sequencer port (default "50051")
+  -listen-all
+    	listen on all network interfaces (0.0.0.0) instead of just localhost
+  -rollup-id string
+    	rollup id (default "rollupId")
+  -batch-time duration
+    	time in seconds to wait before generating a new batch (default 2s)
+  -da_address string
+    	DA address (default "http://localhost:26658")
+  -da_auth_token string
+    	auth token for the DA
+  -da_namespace string
+    	DA namespace where the sequencer submits transactions
+  -db_path string
+    	path to the database
+```
+
+As shown by the help command, a centralized sequencer is configured to serve a rollup (via `rollup_id`). The DA network to persist the sequenced batches are specified using `da_address`, `da_auth_token` and `da_namespace`.
+
+
+## Run the centralized sequencer {#run-the-centralized-sequencer}
+
+:::tip
+As mentioned in the prerequisities, you should have your DA layer running. These steps assume the local-da is running on `http://localhost:7980`.
+:::
+
+Start your centralized sequencer with the following command:
+
+::: code-group
+
+```sh [Quick Start]
+./build/centralized-sequencer -rollup-id my-rollup -da_address http://localhost:7980
+```
+
+```sh [Build a Chain]
+./build/centralized-sequencer -rollup-id wordle -da_address http://localhost:7980
```

Comment:
just realized that we have underscore and dash in flags, when we can have only underscores